lib/catch-test/main.cpp

```diff
@@ -0,0 +1,9 @@
+#define CATCH_CONFIG_RUNNER
+#include "Catch.hpp"
+
+int main(int argc, char* argv[]) {
```

Comment:
How is this different and/or better than:

```
#define CATCH_CONFIG_MAIN
#include "catch.hpp"
```


#define CATCH_CONFIG_MAIN
2 | #include "catch.hpp"
3




#define CATCH_CONFIG_MAIN
2 | #include "catch.hpp"
3




#define CATCH_CONFIG_MAIN
2 | #include "catch.hpp"
3